apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -38,9 +38,155 @@ type DatadogAgentSpec struct {
 }
 
 // DatadogFeatures are Features running on the Agent and Cluster Agent.
-// TODO add the features (APM, Kubernetes State Metrics Core...)
 // +k8s:openapi-gen=true
-type DatadogFeatures struct{}
+type DatadogFeatures struct {
+	// Application-level features
+	// APM (Application Performance Monitoring) configuration
+	APM *APMFeatureConfig `json:"apm,omitempty"`
+	// Logs configuration
+	Logs *LogsFeatureConfig `json:"logs,omitempty"`
```

Comment:
In v1alpha1 we named it `logCollection`. Did we got some feedback to rename it "logs"?